From cff33fbc9c7c69f92be990cef0f19e63ac1f7654 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Sat, 21 Dec 2019 14:46:46 +0100
Subject: [PATCH] Fix build with opencv 4.2

Port away from old API which is removed in 4.2

Differential Revision: https://phabricator.kde.org/D26143
---
 .../facesengine/detection/opencv-face/opencvfacedetector_p.cpp  | 2 +-
 .../facesengine/detection/opencv-face/opencvfacedetector_p.h    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: digikam-6.4.0/core/libs/facesengine/detection/opencvfacedetector.cpp
===================================================================
--- digikam-6.4.0.orig/core/libs/facesengine/detection/opencvfacedetector.cpp
+++ digikam-6.4.0/core/libs/facesengine/detection/opencvfacedetector.cpp
@@ -166,7 +166,7 @@ public:
      * of the region of interest of this cascade (still relative to whole image).
      * For frontal face cascades, returns the given parameter unchanged.
      */
-    cv::Rect faceROI(const CvRect& faceRect) const
+    cv::Rect faceROI(const cv::Rect& faceRect) const
     {
         return cv::Rect(lround(faceRect.x + roi.x()      * faceRect.width),
                         lround(faceRect.y + roi.y()      * faceRect.height),
